.github/workflows/haskell.yml

```diff
@@ -45,15 +45,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        cabal: ["3.6"]
+        cabal: ["3.10"]
         ghc:
-          - "8.4.4"
```

Comment:
`persistent` still supports 8.4 per the cabal file's `base >= 4.11.1.0` bound. I'd prefer to retain support here if we can manage it.